src/connectors/mysql2.ts

```diff
@@ -0,0 +1,47 @@
+import mysql from "mysql2/promise";
+
+import type { Connector, Statement } from "../types";
+
+export default function mysqlConnector(opts: mysql.ConnectionOptions) {
+  let _connection: mysql.Connection | undefined;
+  const getConnection = async () => {
+    if (_connection) {
+      return _connection;
+    }
+
+    _connection = await mysql.createConnection({
```

Comment:
I wan't sure if we want to have some `options` as default, for example we can set the `host` to `localhost` or `password` to `""`...

Reply:
Would be nice to default to root@localhost:3306 but unsure how useful in practice :)

Reply:
yes, or we can set `root@localhost:3306` to default only in dev